src/core/glossary.md

```diff
@@ -0,0 +1,37 @@
+# Glossary
+
+## Simulation Sequence Components
+
+**cache:** Cache is the storage of the solution of a variable from a stage to be used by other stages in the simulation
+
+**chronology:** The vertical inter-stage relationship dictating how variable results impact the next stage's variable parameters.
+- Example: `feedforward_chronologies = Dict(("stage-1" => "stage-2") => Synchronize(periods = 24))`
```

Comment:
add word explanations to the examples. For instance, mention here that this chronology uses the first 24 solutions in the horizon to synchronize with 24 executions of stage 2.